apps/vmq_server/test/vmq_ssl_SUITE.erl

```diff
@@ -103,7 +106,14 @@ init_per_testcase(Case, Config) ->
                                                                {tls_version, "tlsv1.2"},
                                                                {require_certificate, true},
                                                                {crlfile, ssl_path("crl.pem")},
-                                                               {use_identity_as_username, true}])
+                                                               {use_identity_as_username, true}]);
+        {_, _, _, _, _, _, true} ->
+            {ok, _} = vmq_server_cmd:set_config(allow_anonymous, true),
+            {ok, _} = vmq_server_cmd:listener_start(1888, [{ssl, true},
+                                                           {nr_of_acceptors, 5},
+                                                           {psk_support, true},
+                                                           {pskfile, ssl_path("test.psk")},
```

Comment:
@mths1 I suppose you wanted to add that "test.psk" file?

Reply:
Added. And renamed it a little bit to be more "readable"

Reply:
Still fails for me if "it2" is not given in the test.psk file.

 `➜  vernemq git:(2077) ✗ rebar3 ct --suite apps/vmq_server/test/vmq_ssl_SUITE.erl --case connect_psk_wrong_identity`


```
%%% vmq_ssl_SUITE ==> connect_psk_wrong_identity: FAILED
%%% vmq_ssl_SUITE ==> {error,
    {no_matching_error_message,
        {error,
            {tls_alert,
                {illegal_parameter,
                    "TLS client: In state cipher received SERVER ALERT: Fatal - Illegal Parameter\n"}}}}}
```